packages/compiler/formatter/print/printer.ts

```diff
@@ -1080,36 +1083,25 @@ export function printScalarStatement(
 }
 
 export function printNamespaceStatement(
-  path: AstPath<NamespaceStatementNode>,
+  path: AstPath<FlattenedNamespaceStatementNode>,
   options: TypeSpecPrettierOptions,
   print: PrettierChildPrint
 ) {
-  const printNested = (currentPath: AstPath<NamespaceStatementNode>, parentNames: Doc[]): Doc => {
-    const names = [...parentNames, currentPath.call(print, "id")];
-    const currentNode = currentPath.getNode();
-
-    if (
-      !isArray(currentNode?.statements) &&
-      currentNode?.statements?.kind === SyntaxKind.NamespaceStatement
-    ) {
-      return path.call((x) => printNested(x, names), "statements");
-    }
+  const names = path.map(print, "ids");
```

Comment:
It's nice that we get a simplification here in exchange :)

Reply:
haha yeah